app/src/views/QuestionDetailView.vue

```diff
@@ -16,20 +16,49 @@
                 </form>
             </div>
         </div>
-
         <h4 class="mt-5">Answers</h4>
         <p v-if="question.answers && question.answers.length < 1">No answers yet</p>
         <UISelect :options="sortOptions"
                   :value.sync="sortBy"
                   v-if="question.answers && question.answers.length > 0" />
 
+
```

Comment:
No point to duplicate code just to display best answer as first element. 
Better to sort by answer.accepted